package.json

```diff
@@ -51,159 +52,155 @@
+    "eslint-plugin-cypress": "^2.13.3",
+    "eslint-plugin-import": "^2.27.5",
+    "eslint-plugin-jest-dom": "^3.9.4",
+    "eslint-plugin-jsx-a11y": "^6.7.1",
+    "eslint-plugin-no-unsanitized": "^3.2.0",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "events": "~1.1.1",
-    "fast-glob": "^3.2.7",
+    "fast-glob": "^3.2.12",
     "file-loader": "^6.2.0",
-    "fs-extra": "^10.0.0",
-    "husky": "^4.3.6",
+    "fs-extra": "^10.1.0",
+    "husky": "^8.0.0",
     "jest": "^27.5.1",
     "jest-canvas-mock": "^2.5.0",
     "jest-environment-jsdom": "^27.5.1",
     "jest-fetch-mock": "^3.0.3",
-    "jest-styled-components": "^6.3.1",
-    "lint-staged": "^10.5.3",
-    "mockdate": "^2.0.2",
-    "nock": "^13.2.8",
-    "prettier": "^2.1.2",
-    "raf": "^3.4.0",
+    "jest-styled-components": "^6.3.4",
+    "lint-staged": "^11.2.6",
+    "mockdate": "^2.0.5",
+    "nock": "^13.3.1",
+    "prettier": "^2.8.8",
+    "raf": "^3.4.1",
     "react": "^17.0.2",
     "react-dom": "^17.0.2",
     "react-test-renderer": "^17.0.2",
     "rimraf": "^3.0.2",
-    "semantic-release": "^19.0.2",
-    "semver": "^7.3.5",
+    "semantic-release": "^19.0.5",
+    "semver": "^7.5.0",
     "sprintf-js": "^1.1.2",
     "styled-components": "^4.4.1",
-    "ts-node": "^10.4.0",
-    "typescript": "^4.6.2",
+    "typescript": "~4.7.0",
     "typescript-to-proptypes": "^2.2.1",
     "url-loader": "^4.1.1",
     "uuid": "^8.3.2",
-    "webpack-dev-server": "^4.0.0"
+    "webpack-dev-server": "^4.15.0"
   },
   "dependencies": {
-    "@floating-ui/dom": "^1.0.7",
-    "@floating-ui/react-dom": "^1.0.1",
+    "@floating-ui/dom": "^1.2.7",
+    "@floating-ui/react-dom": "^1.3.0",
     "@octokit/rest": "^18.12.0",
     "@styled-system/prop-types": "^5.1.5",
-    "@tanstack/react-virtual": "^3.0.0-beta.39",
-    "@types/styled-system": "^5.1.11",
-    "chalk": "^4.1.1",
-    "ci-info": "^3.3.2",
-    "classnames": "~2.2.6",
+    "@tanstack/react-virtual": "^3.0.0-beta.54",
+    "@types/styled-system": "^5.1.16",
+    "chalk": "^4.1.2",
+    "ci-info": "^3.8.0",
+    "classnames": "~2.3.2",
     "crypto-js": "^4.1.1",
-    "date-fns": "^2.26.0",
+    "date-fns": "^2.30.0",
     "dotenv": "^10.0.0",
     "immutable": "~3.8.2",
     "invariant": "^2.2.4",
-    "lodash": "^4.17.20",
-    "node-fetch": "^3.2.10",
-    "polished": "^4.0.5",
-    "prop-types": "^15.7.2",
+    "lodash": "^4.17.21",
+    "node-fetch": "^3.3.1",
+    "polished": "^4.2.2",
+    "prop-types": "^15.8.1",
     "react-day-picker": "~7.4.10",
-    "react-dnd": "^15.1.1",
-    "react-dnd-html5-backend": "^15.1.2",
+    "react-dnd": "^15.1.2",
+    "react-dnd-html5-backend": "^15.1.3",
     "react-is": "^17.0.2",
-    "react-transition-group": "^4.4.2",
+    "react-transition-group": "^4.4.5",
     "styled-system": "^5.1.5",
-    "wait-on": "^5.2.1"
+    "wait-on": "^5.3.0"
+  },
+  "overrides": {
+    "@types/enzyme": {
+      "@types/react": "$@types/react"
+    }
```

Comment:
`@types/enzyme` uses type declarations for React 16 which clashes with our React 17 typing. This override has been introduced specifically for this nested dependency to workaround this.